### DISCUSSION TITLE
Opinion: The dark theme in the latest commit is a bit too dark

### DISCUSSION THREAD

**Discussion by mz1193** [ubuntu/yaru#2684] [Ideas]
Hi, first of all, thanks for the great work on this theme. I'm not an Ubuntu user, but I've happily been using this on Debian for a few years now and I just love it.

Unfortunately, I'm a little disappointed in the readability of the dark theme in the latest commit. At least on my screen, it seems like it's harder to distinguish between different background elements compared to the "gunmetal dark grey" tone of previous releases, which offered a pleasant dark experience with very clear distinctions between elements.

I understand this is very subjective, but is the new dark color scheme a work in progress or is it set in stone?

Thanks for taking the time to read this. :)

**Comment by Feichtmeier** [comment]
Hi 👋☺️
Nothing is set in stone.
The new darker dark fits the purpose of the dark theme a bit better. Maybe we could tweak the contrast a bit better.
As you probably know we are based on Adwaita and Adwaita dark uses dark borders, which is a rather outdated design approach for dark themes. Flipping those borders to light, like in the new dark menus, is something to explore I think but it could need a lot of fine tuning

**Reply by mz1193** [reply]
Good to know you're open to tweaking the contrast! I do understand the reasoning behind making it darker in the first place, I think what I'm mostly concerned with is, like I said, easier distinction between background elements, especially side panels vs. main window areas.

Thanks for responding so quickly, I look forward to seeing how the theme continues to develop. 🙂

**Comment by mz1193** [comment]
Revisiting this topic after a while, the GTK4 dark contrast is looking a lot better than it used to, kudos for that. However I'm still wondering if there is any interest in getting the GTK3 dark theme to look closer to how it was back around Ubuntu 20.04, which was slightly lighter and warmer and, imo, easier on the eye than the current rendition.
